lib/board.rb

```diff
@@ -0,0 +1,36 @@
+require_relative './prompt.rb'
+# board class
+class Board
+  include Prompt
+  def initialize
+    @state = Array.new(9, '·')
+    @cursor = 4
+  end
+
+  def print(player_name = 'Test')
+    # prompt = TTY::Prompt.new(Interrupt: :exit)
+    board = [
+      "                   ",
+      "  ┏━━━━━━━━━━━━━┓  ",
+      "  ┃    ╷   ╷    ┃  ",
+      "  ┃ #{pp(0)}│#{pp(1)}│#{pp(2)} ┃  ",
+      "  ┃╶───┼───┼───╴┃  ",
+      "  ┃ #{pp(3)}│#{pp(4)}│#{pp(5)} ┃  ",
+      "  ┃╶───┼───┼───╴┃  ",
+      "  ┃ #{pp(6)}│#{pp(7)}│#{pp(8)} ┃  ",
+      "  ┃    ╵   ╵    ┃  ",
+      "  ┗━━━━━━━━━━━━━┛  ",
+      "                   "
+    ].join("\n")
+    system 'clear'
```

Comment:
Let's make this even looser and independent, let's not know about the existence of prompt and move these methods outside of Board class. To the game flow for example.